src/nv_ingest/framework/orchestration/ray/util/pipeline/pipeline_builders.py

```diff
@@ -47,16 +48,114 @@ def export_config_to_env(ingest_config: Any) -> None:
+    - RAY_LOGGING_ROTATE_BACKUP_COUNT: Number of backup log files to keep. Default: 19
+    - RAY_DISABLE_IMPORT_WARNING: Disable Ray import warnings (0/1). Default: 0
+    - RAY_USAGE_STATS_ENABLED: Enable/disable usage stats collection (0/1). Default: 1
+    """
+    # Get log level from environment, default to INFO
+    log_level = os.environ.get("RAY_LOGGING_LEVEL", "INFO").upper()
+
+    # Validate log level
+    valid_levels = ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
+    if log_level not in valid_levels:
+        logger.warning(f"Invalid RAY_LOGGING_LEVEL '{log_level}', using INFO. Valid levels: {valid_levels}")
+        log_level = "INFO"
+
+    # Get encoding format from environment, default to TEXT
+    encoding = os.environ.get("RAY_LOGGING_ENCODING", "TEXT").upper()
+
+    # Validate encoding
+    valid_encodings = ["TEXT", "JSON"]
+    if encoding not in valid_encodings:
+        logger.warning(f"Invalid RAY_LOGGING_ENCODING '{encoding}', using TEXT. Valid encodings: {valid_encodings}")
+        encoding = "TEXT"
+
+    # Get additional standard logger attributes
+    additional_attrs_str = os.environ.get("RAY_LOGGING_ADDITIONAL_ATTRS", "")
+    additional_log_standard_attrs = []
+    if additional_attrs_str:
+        additional_log_standard_attrs = [attr.strip() for attr in additional_attrs_str.split(",") if attr.strip()]
+
+    # Set log deduplication environment variable if specified
+    dedup_logs = os.environ.get("RAY_DEDUP_LOGS", True)
+    if dedup_logs is not None:
+        os.environ["RAY_DEDUP_LOGS"] = str(dedup_logs)
+
+    # Set log to driver environment variable if specified
+    log_to_driver = os.environ.get("RAY_LOG_TO_DRIVER", True)
+    if log_to_driver is not None:
+        os.environ["RAY_LOG_TO_DRIVER"] = str(log_to_driver).lower()
+
+    # Configure log rotation settings
+    rotate_bytes = os.environ.get("RAY_LOGGING_ROTATE_BYTES", "1073741824")  # Default: 1GB per file
+    if rotate_bytes is not None:
+        try:
+            rotate_bytes_int = int(rotate_bytes)
+            os.environ["RAY_LOGGING_ROTATE_BYTES"] = str(rotate_bytes_int)
+        except ValueError:
+            logger.warning(f"Invalid RAY_LOGGING_ROTATE_BYTES '{rotate_bytes}', using default (1GB)")
+            os.environ["RAY_LOGGING_ROTATE_BYTES"] = "1073741824"
+
+    rotate_backup_count = os.environ.get("RAY_LOGGING_ROTATE_BACKUP_COUNT", "19")  # Default: 19 backups (20GB Max)
+    if rotate_backup_count is not None:
+        try:
+            backup_count_int = int(rotate_backup_count)
+            os.environ["RAY_LOGGING_ROTATE_BACKUP_COUNT"] = str(backup_count_int)
+        except ValueError:
+            logger.warning(f"Invalid RAY_LOGGING_ROTATE_BACKUP_COUNT '{rotate_backup_count}', using default (19)")
+            os.environ["RAY_LOGGING_ROTATE_BACKUP_COUNT"] = "19"
+
+    # Configure Ray internal logging verbosity
+    disable_import_warning = os.environ.get("RAY_DISABLE_IMPORT_WARNING")
```

Comment:
Should we add a default value here, e.g., `os.environ.get("RAY_DISABLE_IMPORT_WARNING", 0)` because as written I don't think lines 119-122 do anything.